docs/source/getting_started/notebooks.md

```diff
@@ -0,0 +1,11 @@
+# Llama Stack Notebooks
+
+In these sets of notebooks, we will walk you through the main sets of APIs we offer with Llama Stack with working examples to explore the possibilities that Llama Stack opens up for you.
+
+Please use the following Google Colab links to explore the notebooks:
+
+## Table of Contents
+1. [Llama Stack Showcase](https://colab.research.google.com/drive/1F2ksmkoGQPa4pzRjMOE6BXWeOxWFIW6n?usp=sharing): How you can build LLM-powered agentic applications using Llama Stack.
```

Comment:
@raghotham do you have any suggestions other than "Showcase" as a name for this notebook? Everything here is a showcase so I don't know. Should this be the new "Getting Started" notebook?

Reply:
I named it to Llama Stack Building AI Application for now. Let me know if there's a better suggestion!